external-api/src/v1/controllers/party-db.controller.ts

```diff
@@ -21,7 +21,9 @@ export const lookup = async (req: Request, res: Response) => {
 
   const response: { status: number; data: unknown } = await axios({
     method: 'get',
-    url: `${APIM_MDM_URL}customers?companyReg=${companyReg}`,
+    url: AUTOMATIC_SF_CUSTOMER_CREATION_ENABLED
+      ? `${APIM_MDM_URL}customers/direct?companyRegistrationNumber=${companyReg}`
```

Comment:
This uses the new direct SF api in MDM, old route is Informatica

Reply:
Could we call this route `customers/salesforce` rather than `customers/direct`, makes it a little clearer/less tied to the fact that an endpoint already exists

Reply:
Sure, I agree /salesforce is slightly clearer, will update my MDM work to match also